### ISSUE TITLE
Help! Encounter menu is broken

### ISSUE THREAD

**Issue by tigerflavoredcereal** [rpgboss/rpgboss#349]
Hey there! I've spent a good part of the afternoon messing around with RPGBoss, and for the most part it's working really well. However ... 
![screen shot 2015-04-07 at 6 48 35 pm](https://cloud.githubusercontent.com/assets/11845460/7037430/a002fdf8-dd57-11e4-82cf-24b45df104fe.png)
... the encounter screen is bugged out. Has anyone else had this problem?

**Comment by ravedragon** [comment]
is this linux? or mac, try to resize the database window, close/reopen database.
it might be trying to load takes a moment.

**Comment by cartoonzx** [comment]
This happens on windows as well. A lot actually.

**Comment by hendrik-weiler** [comment]
Its always on mac.

**Comment by cartoonzx** [comment]
Something similar happens on Windows.

**Comment by tommycli** [comment]
Yeah this happens to me ocassionally also. Is there any error message in the console you guys have?

**Comment by PyromaniacalKitten** [comment]
When you click 'okay', and the click 'database' again, the big black box usually disappears for me. then go straight to encounters, without clicking on any other boxes.
